supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -194,7 +194,7 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor,           // W
     val definitionArn = createDefinition(workflow.callable.name)
 
     val job = client.submitJob(SubmitJobRequest.builder()
-                .jobName(sanitize(workflow.callable.name))
+                .jobName(sanitize(s"""${workflow.callable.name}-${jobDescriptor.taskCall.callable.name}"""))
```

Comment:
The callable name isn't necessary unique per call, eg
```
call my_task as a
call my_task as b
```

The callable name is `my_task` both times, you probably want the call name

Reply:
Confused. Do you mean `workflow.callable.name` is not unique or `${workflow.callable.name}-${jobDescriptor.taskCall.callable.name}` is not unique?

Reply:
I would suggest `.jobName(sanitize(jobDescriptor.taskCall.fullyQualifiedName))` to replace the workflow and task names here, since that will be filled in with the full path to the call (but double check that it gives you something sensible).

Eg if this is a call to a task 5 subworkflows deep you want the full context not just the current workflow and the current task name.